packages/core/src/destination-kit/index.ts

```diff
@@ -94,12 +95,15 @@ export type AudienceResult = {
 }
 
 export type AudienceMode = { type: 'realtime' } | { type: 'synced'; full_audience_sync: boolean }
+export type Personas = { computation_id: 'string'; computation_key: 'string' } & { [key: string]: any }
```

Comment:
why not?
```suggestion
export type Personas = { computation_id: 'string'; computation_key: 'string', { [key: string]: any }}
```
whats the need for intersecting types here? `&` can be useful if you are combining two predefined types and in this step, we are creating a plain new type.